frontends/web/src/routes/exchanges/exchanges.tsx

```diff
@@ -62,23 +62,19 @@ class Exchanges extends Component<Props, State> {
         this.setState(({ region }) => ({ region: region !== code ? code : null }));
     }
 
-    private toggleMethods = method => {
-        this.setState(({ methods }) => ({
-            methods: methods.includes(method)
-                ? methods.filter(m => m !== method)
-                : [method, ...methods],
-        }));
+    private toggleMethod = code => {
+        this.setState(({ method }) => ({ method: method !== code ? code : null }));
     }
 
     public render(
         { t }: RenderableProps<Props>,
         {
-        methods,
+        method,
         region,
     }) {
         const results = this.data
             .filter(({ regions }) => !region || regions.includes(region))
-            .filter(({ payment }) => !methods.length || methods.some(m => payment.includes(m)))
+            .filter(({ payment }) => !method || payment.includes(method))
```

Comment:
shouldn't this be `... || payment === method` now?

Reply:
`region` and `method` are string to filter from, but 
`regions` and `payment` are arrays, see https://github.com/digitalbitbox/bitbox-wallet-app/blob/f42efa2f6962de963af8e4b7d180aa3dfe391300/frontends/web/src/routes/exchanges/exchanges-data.ts#L8-L9

Reply:
Should I rename `payment ` to be `payments` ?

Reply:
Oh, my bad. I think it's not worth renaming. :+1: